src/Sql/dbo/Tables/OrganizationPasswordManager.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE [dbo].[OrganizationPasswordManager](
+    [Id] [uniqueidentifier] NOT NULL,
+    [OrganizationId] [uniqueidentifier] NOT NULL,
+    [Plan] [nvarchar](50) NOT NULL,
+    [PlanType] [tinyint] NOT NULL,
+    [Seats] [int] NULL,
+    [MaxCollections] [smallint] NULL,
+    [UseTotp] [bit] NULL,
+    [UsersGetPremium] [bit] NOT NULL,
+    [Storage] [bigint] NULL,
+    [MaxStorageGb] [smallint] NULL,
+    [MaxAutoscaleSeats] [int] NULL,
+    [RevisionDate] DATETIME NULL
+) 
```

Comment:
We should add some indexes here. The `Organization` table has the following indexes which we should probably follow.

```tsql
CONSTRAINT [PK_OrganizationPasswordManager] PRIMARY KEY CLUSTERED ([Id] ASC)
```